### ISSUE TITLE
test new DBS Client v 4.0.0

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#6851]
from Valentin:
```
Hi,
finally, I managed to merge full set of changes (from DBSClient4go branch)
into main dmwm/DBSClient one. I also created a new tag 4.0.0 which separates
previous DBSClient versions/tags (3.X.Y) which were working only with Python
based DBS server. This will allow us easily switch between the two if necessary.
If anything is missing (or we find some bugs/flaws) I committed to give high
priority to DBSClient to fix these issues.

With new tag I think we are ready for the next step of DBS Migration which
consists of the following action items:
- write to CMS HN an announcement email about upcoming changes
  - I already gave Kevin my first and second draft and awaiting his reply
- WMCore team can start applying new DBSClient in all necessary places
- CRAB team can do the same
- once new DBSClient is in place I'll ask both WMCore and CRAB team to use it
to access their favorite data on testbed via the following setup:

from dbs.apis.dbsClient import DbsApi
url = "https://cmsweb-testbed.cern.ch/dbs2go"
api = DbsApi(url=url, debug=1)
# use any API for data access, e.g.
res = api.listDataTiers()
print(res)

Please let me know or open up DBSClient github issue and assign to me if you'll
find any misbehavior.

Best,
Valentin.
```

**Comment by belforte** [comment]
I ran a small test "by hand" but this requires python3 so we can do more extensive tests only after we move TaskWorker and then Publisher to python3.
Until then, if one wants to test by hand in standalone mode, can follow the instructions in the DBSClient wiki in GH. But also I have done the needed thing on the crab-prod-tw01 VM (not in the container!) and one can simply run there.

**Comment by belforte** [comment]
On HOLD until we have a TW which runs with python3, then will look into adding the new DBS Client.

**Comment by mapellidario** [comment]
The images (both crabserver and crabaskworker) built with py3 env from now on all have the new dbs client. The first such images are the ones with the tag `py3.211215`.

Rerefences:
- https://github.com/dmwm/CRABServer/releases/tag/py3.211215
- https://github.com/cms-sw/cmsdist/pull/7511
- https://github.com/cms-sw/cmsdist/pull/7508

**Comment by belforte** [comment]
now running on test instance (Publisher_sched crab-dev-tw01). Simple publications worked fine.

**Comment by belforte** [comment]
note DBS v4.0.5. is out https://github.com/cms-sw/cmsdist/pull/7524#issuecomment-997700166

**Comment by belforte** [comment]
new  client is now part of the build
@mapellidario please add a note here with the version used in latrest images

**Comment by belforte** [comment]
@mapellidario we are now routinely running with new client in preprod and test instances. So I think this is done. If you agree, please close.

**Comment by mapellidario** [comment]
I do agree, closing with this comment
